Unable to install on node 16.

node -  16.13.1
npm - 8.3.0

Hitting an error when trying to install this on node 16. 
Had no issues with node 14.

```
me@radmins-MacBook-Pro % npm i laravel-mix-simple-image-processing --save-dev
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm ERR! code 1
npm ERR! path /.../node_modules/sharp
npm ERR! command failed
npm ERR! command sh -c (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)
npm ERR! TOUCH Release/obj.target/libvips-cpp.stamp
npm ERR!   CXX(target) Release/obj.target/sharp/src/common.o
```

Based on the error, I'm assuming it's the sharp version?
https://www.npmjs.com/package/sharp

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
     "imagemin-pngquant": "^8.0.0",
     "imagemin-webp": "^5.1.0",
     "path": "^0.12.7",
-    "sharp": "^0.24.0"
+    "sharp": "^0.29.3"
   },
   "scripts": {
     "test": "echo \"No test specified\" && exit 0"

